### PR TITLE
Guardian sms templates

### PR DESCRIPTION
**Pull request by joseluisdiaz**
guardian sms templates

### CODE REVIEW

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+![](/media/articles/mfa/sms-config.png)
+
+You have two text area to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+ 
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the currently supported templating syntax to use when accessing user attributes in your SMS templates. Here are the attributes available to you:
+* `code`: The Enrollment/Verification code. 
+* `requestInfo.lang`: The browser language.
+* `tenant.friendlyName`: Your **Friendly Name** setted in [Account Settings](${manage_url}/#/account), scrolling down to _Settings_.
```
Comment (dafortune) [inline articles/multifactor-authentication/administrator/sms-templates.md]: set is irregular I think `set` `set` `set`

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two text area to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+ 
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the currently supported templating syntax to use when accessing user attributes in your SMS templates. Here are the attributes available to you:
```
Comment (dafortune) [inline articles/multifactor-authentication/administrator/sms-templates.md]: Sounds like if it says `syntax` too many times to me 😄 my be good, I'm not native, of course.

Comment (dafortune) [reply articles/multifactor-authentication/administrator/sms-templates.md]: Also I don't see any user attribute in the list (code, tenant friendly name).... maybe lang could be consider so?

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+description: Customize SMS Messages
+---
+
+# Customize SMS Messages
+
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
```
Comment (nikolaseu) [inline articles/multifactor-authentication/administrator/sms-templates.md]: > ... or verify (when you proof that have that device).

this doesn't sound right

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two text area to customize your messages:
```
Comment (nikolaseu) [inline articles/multifactor-authentication/administrator/sms-templates.md]: `areas` ?

but I don't know if it's ok to say "text areas", maybe just "fields" ?

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+ 
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the currently supported templating engine to use when accessing user attributes in your SMS templates. Here are the attributes available to you:
+* `code`: The Enrollment/Verification code. 
+* `requestInfo.lang`: The browser language.
+* `tenant.friendlyName`: Your **Friendly Name** set in [Account Settings](${manage_url}/#/account), scrolling down to _Settings_.
```
Comment (nikolaseu) [inline articles/multifactor-authentication/administrator/sms-templates.md]: > ... scrolling down to _Settings_.

I'd rather not be that specific about the location inside the page

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two text area to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+ 
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the currently supported templating engine to use when accessing user attributes in your SMS templates. Here are the attributes available to you:
```
Comment (nikolaseu) [inline articles/multifactor-authentication/administrator/sms-templates.md]: Can this whole phrase be simplified?

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+* `code`: The Enrollment/Verification code. 
+* `requestInfo.lang`: The browser language.
```
Comment (nikolaseu) [inline articles/multifactor-authentication/administrator/sms-templates.md]: In which format? just add an example

**Review by nikolaseu — COMMENTED** [review]
Is it ok to change the "style" of the screenshot? I think they use full browser screenshots everywhere

File: articles/multifactor-authentication/administrator/sms-templates.md (hunk 3 above)
Comment (topherMarie) [reply articles/multifactor-authentication/administrator/sms-templates.md]: To customize the content of the messages sent by Auth0 during enrollment (when associating a device to Guardian) or verification (when an authentication messages is sent to the device), do the following.

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+# Customize SMS Messages
+
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two fields to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number.
```
Comment (topherMarie) [inline articles/multifactor-authentication/administrator/sms-templates.md]: * Enrollment Template: the message sent by Auth0 during device enrollment.

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two fields to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone.
```
Comment (topherMarie) [inline articles/multifactor-authentication/administrator/sms-templates.md]: Verification Template: the message sent by Auth0 to verify the possession of the device.

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two fields to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+
```
Comment (topherMarie) [inline articles/multifactor-authentication/administrator/sms-templates.md]: [Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the supported templating engine to use when accessing user attributes in SMS templates. The following attributes are available:

File: articles/multifactor-authentication/administrator/sms-templates.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+![](/media/articles/mfa/sms-config.png)
+
+You have two fields to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
+ 
+[Liquid](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) syntax is the currently supported templating engine to use when accessing user attributes in your SMS templates. Here are the attributes available to you:
+* `code`: The Enrollment/Verification code. 
+* `requestInfo.lang`: The browser language (ie, `es-AR,es;q=0.8`, `en-US,en`, etc.).
+* `tenant.friendlyName`: Your **Friendly Name** set in [Account Settings](${manage_url}/#/account).
```
Comment (topherMarie) [inline articles/multifactor-authentication/administrator/sms-templates.md]: `tenant.friendlyName`: The **Friendly Name** set in [Account Settings](${manage_url}/#/account).

**Pull request merged**